### PR TITLE
 convert some HOC download strings to markdown so we can render them safely

### PR DESCRIPTION
**Pull request by Hamms**

### CODE REVIEW

File: dashboard/config/locales/en.yml [hunk 1]
```diff
@@ -967,9 +967,9 @@ en:
   hoc_download:
     title: 'Hour of Code downloads'
     capsule_title: 'Do you have Internet?'
-    capsule_desc: 'Use the online, web-based <a href="%{url}">%{app_name}</a>.'
+    capsule_desc: 'Use the online, web-based %{app_name}](%{url}).'
```
Comment (islemaster) [inline dashboard/config/locales/en.yml]:
```suggestion
    capsule_desc: 'Use the online, web-based [%{app_name}](%{url}).'
```

Comment (Hamms) [reply dashboard/config/locales/en.yml]: d'oh! Thanks

**Review by islemaster — APPROVED** [review]

**Comment by joshlory** [comment]
Possible regression from this change: https://codedotorg.slack.com/archives/CFTFD6BPV/p1565136039092900

image

**Comment by Hamms** [comment]
ahhh, the problem is that those are invalid urls. I'll fix

**Pull request merged**